app/src/main/java/net/christianbeier/droidvnc_ng/Defaults.kt

```diff
@@ -68,6 +72,7 @@ class Defaults {
             this.portRepeater = readDefault.portRepeater
             this.scaling = readDefault.scaling
             this.password = readDefault.password
+            this.removePadding = readDefault.removePadding
```

Comment:
If you could also update the example JSON in the README, that'd be awesome :-)

Reply:
Added in ba78bb6